powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/ActivationServiceBehavior.java

```diff
@@ -105,6 +110,11 @@ public void setActivationHistoryServiceBehavior(ActivationHistoryServiceBehavior
         this.activationHistoryServiceBehavior = activationHistoryServiceBehavior;
     }
 
+    @Autowired
```

Comment:
Why do we use setter injection here? Could we use constructor injection that we already use in this class?

Reply:
There are more fields in this class that use setter injection than fields using constructor injection, so I followed this pattern.